### PR TITLE
Shellcheck compliancy

### PR DESCRIPTION
**Pull request by AuditeMarlow**
This tackles #23 pretty nicely I suppose.

### CODE REVIEW

**Comment by AuditeMarlow** [comment]
I had only just now notice that you (@theova) also ave a branch like this ready in your `base16-manager` fork. Er... Whoops!

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+sudo: required
```
Comment (theova) [inline .travis.yml:1]: Why should `sudo` be required?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+language: bash
+
+services:
+  - docker
+
+before_install:
+  - docker pull koalaman/shellcheck
+
+script:
+  - docker run -v $(pwd):/scripts koalaman/shellcheck /scripts/base16-manager
```
Comment (theova) [inline .travis.yml:12]: `shellcheck` is officially supported by travis. Therefore, we don't need to install it manually.
Just run
`-  shellcheck base16-manager`

**Review by theova — COMMENTED** [review]
Very good job!

I'm sorry, that the work is done twice, in future, I'll always branch out in this repository.

I've add some more small changes conerning shellcheck, the file `$CONFIG_PATH/installed` and functions description.

File: .travis.yml (hunk 1 above)
Comment (AuditeMarlow) [reply .travis.yml:1]: Docker is normally not available to a normal user unless it is added to the `docker` group, hence why TravisCI needs `sudo`.

File: .travis.yml (hunk 2 above)
Comment (AuditeMarlow) [reply .travis.yml:12]: I am aware of this, but am following the trend of containerizing everything. I prefer running ShellCheck from the official Docker image. :)

**Pull request merged**